examples/config/go.mod

```diff
@@ -0,0 +1,54 @@
+module go.opentelemetry.io/contrib/examples/config
+
+go 1.23.6
```

Comment:
This needs to support 1.22 for another release. It needs to support 1.23.0 after that.